docs/drafts/version-ranges.md

```diff
@@ -0,0 +1,104 @@
+# Operator version ranges
+
+This document explains how to specify a version range to install or update an Operator with OLM 1.0.
+
+You define an Operator's target version in its custom resource (CR) file.
+
+## Not specifying a version in the CR
+
+If you do not specify a version in the Operator's CR, OLM 1.0 installs or updates the latest version of the Operator that can be resolved.
+The resolved version is the latest version of the Operator that satisfies the dependencies and constraints of the Operator and the environment.
+Updates that can be resolved are applied automatically when they are published to the catalog.
+
+For more information about dependency resolution in OLM 1.0, see the [Deppy introduction](https://github.com/operator-framework/deppy#introductionhttps://github.com/operator-framework/deppy#introductionhttps://github.com/operator-framework/deppy#introduction)
+
+## Specifying a version in the CR
+
+If you specify a version in the Operator's CR, OLM 1.0 installs or updates the specified version.
+Operator updates are not applied automatically.
+If you want to update the Operator, you must manually edit the CR and apply the changes to the cluster.
+
+## Specifying a version range in the CR
+
+If you specify a version range in the Operator's CR, OLM 1.0 installs or updates the latest version of the Operator that can be resolved within the version range.
+Operator updates within the specified range are automatically installed if they can be resolved successfully.
+Updates are not installed if they are outside of the specified range or if they cannot be resolved successfully.
+
+### Comparisons
+
+A comparison string is composed of a list of comma or space separated values and one or more operators. You can add an additional comparison string by including an OR (`||`) operator between the strings.
+
+#### Basic comparisons
+
+| Operator | Definition                        |
+|----------|-----------------------------------|
+| `=`      | equal (not aliased to an operator |
+| `!=`     | not equal                         |
+| `>`      | greater than                      |
+| `<`      | less than                         |
+| `>=`     | greater than or equal to          |
+| `<=`     | less than or equal to             |
+
```

Comment:
I created https://github.com/operator-framework/operator-controller/issues/550 to further explain[ working with pre-releases](https://github.com/Masterminds/semver/blob/2f39fdc11c33c38e8b8b15b1f04334ba84e751f2/README.md#working-with-prerelease-versions)